frontend/wallet/src/render/views/settings/SettingsPage.re

```diff
@@ -81,14 +81,13 @@ module SettingsQuery = ReasonApollo.CreateQuery(SettingsQueryString);
 module WalletItem = {
   [@react.component]
   let make = (~publicKey) => {
-    let (addressBook, _) = React.useContext(AddressBookProvider.context);
     let keyStr = PublicKey.toString(publicKey);
     let route = "/settings/" ++ Js.Global.encodeURIComponent(keyStr);
     <div
       className=Styles.walletItem
       onClick={_ => ReasonReact.Router.push(route)}>
       <div className=Styles.walletName>
-        {React.string(AddressBook.getWalletName(addressBook, publicKey))}
+        <WalletName pubkey={publicKey} />
```

Comment:
maybe we can rename the prop in <WalletName> publicKey so we can use shorthand notation?

Reply:
We've decided to rename all publicKey to pubkey

Reply:
ok whatever name we want, we can be consistent 
`let make = (~pubkey) => {`

Reply:
:'(